<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>772</width>
    <height>586</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="minimumSize">
    <size>
     <width>732</width>
     <height>527</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>732</width>
     <height>527</height>
    </size>
   </property>
   <layout class="QGridLayout" name="gridLayout_2" columnstretch="0,0">
    <property name="sizeConstraint">
     <enum>QLayout::SizeConstraint::SetFixedSize</enum>
    </property>
    <item row="0" column="1">
     <widget class="QWidget" name="plotWidget" native="true"/>
    </item>
    <item row="0" column="0">
     <widget class="QWidget" name="widget_5" native="true">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QWidget" name="widget" native="true">
         <property name="styleSheet">
          <string notr="true"/>
         </property>
         <layout class="QGridLayout" name="gridLayout">
          <item row="6" column="0">
           <widget class="QPushButton" name="pushButton_13">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>1</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">digitsButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QPushButton" name="pushButton_17">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #808080;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>(</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">operatorsButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QPushButton" name="pushButton_7">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>9</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">digitsButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QPushButton" name="pushButton_18">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>0</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">digitsButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QPushButton" name="pushButton_19">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #808080;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>)</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">operatorsButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="6" column="3">
           <widget class="QPushButton" name="pushButton_16">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #808080;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>+</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">operatorsButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QPushButton" name="pushButton_11">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>6</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">digitsButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="4" column="0">
           <widget class="QPushButton" name="pushButton_5">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>7</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">digitsButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QPushButton" name="XButton">
            <property name="font">
             <font>
              <family>Segoe UI</family>
              <pointsize>12</pointsize>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #808080;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QPushButton" name="pushButton_8">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #808080;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>*</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">operatorsButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QPushButton" name="pushButton_6">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>8</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">digitsButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QPushButton" name="pushButton_4">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #808080;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>/</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">operatorsButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="2" column="3">
           <widget class="QPushButton" name="ACButton">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: rgb(255, 136, 0);
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	border: 1px solid gray;
	background-color: rgb(250, 146, 0);
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>AC</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QPushButton" name="pushButton_2">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #808080;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>sqrt</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">functionsButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="5" column="3">
           <widget class="QPushButton" name="pushButton_12">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #808080;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>-</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">operatorsButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="pushButton_3">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #808080;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>^</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">operatorsButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="7" column="3">
           <widget class="QPushButton" name="enterButton">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="font">
             <font>
              <pointsize>18</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: rgb(0, 174, 255);
	border: 1px solid gray;
	color: black;
}

QPushButton:pressed {
	border: 1px solid gray;
	background-color: rgba(0, 148, 216, 252);
	color: black;
}</string>
            </property>
            <property name="text">
             <string>↵</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QPushButton" name="pushButton_9">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>4</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">digitsButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="0" column="0" colspan="4">
           <widget class="QLineEdit" name="lineEdit">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>0</height>
             </size>
            </property>
            <property name="font">
             <font>
              <pointsize>18</pointsize>
              <italic>true</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="cursor">
             <cursorShape>IBeamCursor</cursorShape>
            </property>
            <property name="styleSheet">
             <string notr="true">QLineEdit {
	border: 1px solid gray;
	color: white;
	background-color: #050505;
	padding: 5px;
}</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignCenter</set>
            </property>
            <property name="placeholderText">
             <string>F(X)</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QPushButton" name="pushButton_10">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>5</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">digitsButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QPushButton" name="pushButton_14">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>2</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">digitsButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QPushButton" name="pushButton">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #808080;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>log10</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">functionsButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QPushButton" name="pushButton_15">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #A9A9A9;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>3</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">digitsButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QPushButton" name="pushButton_20">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #808080;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>.</string>
            </property>
            <attribute name="buttonGroup">
             <string notr="true">operatorsButtonGroup</string>
            </attribute>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QPushButton" name="backSpaceButton">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <italic>false</italic>
              <bold>true</bold>
             </font>
            </property>
            <property name="styleSheet">
             <string notr="true">QPushButton {
	background-color: #808080;
	border: 1px solid gray;
	padding: 5px;
}

QPushButton:pressed {
	background-color: #707070;
	border: 1px solid gray;
	padding: 5px;
}</string>
            </property>
            <property name="text">
             <string>⌫</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_3" native="true">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetNoConstraint</enum>
          </property>
          <item>
           <widget class="QLabel" name="label">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Min X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="minSpinBox">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="minimum">
             <number>-10000</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item>
        <widget class="QWidget" name="widget_4" native="true">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="font">
             <font>
              <pointsize>12</pointsize>
              <bold>true</bold>
             </font>
            </property>
            <property name="text">
             <string>Max X</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="maxSpinBox">
            <property name="font">
             <font>
              <pointsize>10</pointsize>
             </font>
            </property>
            <property name="minimum">
             <number>-10000</number>
            </property>
            <property name="maximum">
             <number>10000</number>
            </property>
            <property name="value">
             <number>500</number>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>772</width>
     <height>33</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar">
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="functionsButtonGroup"/>
  <buttongroup name="digitsButtonGroup"/>
  <buttongroup name="operatorsButtonGroup"/>
 </buttongroups>
</ui>
